File: SpringBoot/src/main/java/com/mycompany/rentCar/Entities/Reclamation.java
package com.mycompany.rentCar.Entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@Entity
@AllArgsConstructor
@NoArgsConstructor
public class Reclamation {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    private String description;
    private Long userId;
    private Long agencyId;

}

File: Angular/src/app/Models/reclamation.ts
export class ListReclamation {

     id?: number;
     name?: string;
     description? : string;
     userId ? : number;
     agencyId ? : number;
    // constructor(
    //   public this.id: number,
    //   public name: string,
    //   public description : string,
    //   public userId ? : number,
    //   public agencyId ? : number
      
     
   
    // ) {}
    constructor(id?: number, name?: string, description?: string, userId?: number, agencyId?: number) {
      this.id = id;
      this.name = name;
      this.description = description;
      this.userId = userId;
      this.agencyId = agencyId;
    }

  }